pytorch_lightning/strategies/ddp.py

```diff
@@ -63,7 +63,8 @@
 if _FAIRSCALE_AVAILABLE:
     from fairscale.optim import OSS
 if _TORCH_GREATER_EQUAL_1_10:
```

Comment:
This is simpler maybe?
```suggestion
if _TORCH_GREATER_EQUAL_1_10 and torch.distributed.is_available():
```

Reply:
Ah, smart!!! @akihironitta - Thanks!